File: PYTHON/Assignment 3/Question 1.py
''' Given a list of integers, compute two sums: one for all even 
numbers, and one for all odd numbers. Then print both sums.'''

# Input: number of integers to process
n = int(input("Enter any number: "))

# Initialize sum for even numbers
sum_even = 0

# Initialize sum for odd numbers
sum_odd = 0

# Loop through n iterations to get integers
for i in range(n):
    # Input: get each number from user
    num = int(input("Enter n: "))
    
    # Check if number is even (divisible by 2)
    if num % 2 == 0:
        # Add to even sum
        sum_even += num
    else:
        # Add to odd sum
        sum_odd += num

# Print both sums: even sum first, then odd sum (space separated)
print(sum_even, sum_odd)

File: JAVA/Practice/Question1.java
class Fibonacci {
    void printFirstTen() {
        int a = 0;
        int b = 1;
        int sum = 0;

        System.out.println("First 10 Fibonacci numbers:");
        for (int i = 1; i <= 10; i++) {
            System.out.print(a + (i < 10 ? " " : ""));
            sum += a;
            int next = a + b;
            a = b;
            b = next;
        }

        System.out.println();
        System.out.println("Sum of the first 10 Fibonacci numbers: " + sum);
    }
}

public class Question1 {
    public static void main(String[] args) {
        Fibonacci fibonacci = new Fibonacci();
        fibonacci.printFirstTen();
    }
}
